apps/synapse-portal-framework/src/components/Explore/ExploreWrapperTabs.tsx

```diff
@@ -36,12 +37,17 @@ export function ExploreWrapperTabs(props: ExploreWrapperProps) {
   const navigate = useNavigate()
   const isMobileView = useMediaQuery(theme.breakpoints.down('sm'))
 
+  const pathnameWithoutTrailingSlash = useMemo(
+    () => pathname.replace(/\/$/, ''),
+    [pathname],
+  )
+
   /**
    * In the desktop view, we use Material UI tabs
    */
   return (
     <Tabs
-      value={pathname}
+      value={pathnameWithoutTrailingSlash}
```

Comment:
unrelated bug I happened to catch on explore pages - if the path is `/Explore/Datasets`, the tab matched, but not if it is `/Explore/Datasets/` -- if there is a trailing slash, remove it.